src/main/view/game/TechOverviewPanel.java

```diff
@@ -2,28 +2,223 @@
+			pesticides.setIcon(getAssets().getImage("ICON_FOOD_HUMAN"));
+		} else {
+			fertilizer.setIcon(getAssets().getImage("ICON_FOOD_PANDA")); 
+			irrigation.setIcon(getAssets().getImage("ICON_FOOD_PANDA"));
+			pesticides.setIcon(getAssets().getImage("ICON_FOOD_PANDA"));
+		}
+	}
 
+	private void drawConnectors() {
+		//These are all those little connector graphics in between the technologies
+		techGraphics.drawImage(getAssets().getImage("TECH_CONNECT1"), 235, 132);
+		techGraphics.drawImage(getAssets().getImage("TECH_CONNECT2"), 235, 289);
+		techGraphics.drawImage(getAssets().getImage("TECH_CONNECT1"), 510, 180);
+		techGraphics.drawImage(getAssets().getImage("TECH_CONNECT3"), 785, 124);
+		techGraphics.drawImage(getAssets().getImage("TECH_CONNECT5"), 1060, 124);
+		techGraphics.drawImage(getAssets().getImage("TECH_CONNECT5"), 1060, 235);
+		techGraphics.drawImage(getAssets().getImage("TECH_CONNECT5"), 1060, 345);
+		techGraphics.drawImage(getAssets().getImage("TECH_CONNECT5"), 1335, 125);
+		techGraphics.drawImage(getAssets().getImage("TECH_CONNECT5"), 1335, 235);
+		techGraphics.drawImage(getAssets().getImage("TECH_CONNECT5"), 1335, 345);
+		techGraphics.drawImage(getAssets().getImage("TECH_CONNECT1"), 1610, 235);
+		techGraphics.drawImage(getAssets().getImage("TECH_CONNECT4"), 1610, 125);
+		techGraphics.drawImage(getAssets().getImage("TECH_CONNECT6"), 1885, 180);
+		techGraphics.drawImage(getAssets().getImage("TECH_CONNECT5"), 2160, 180);
+		techGraphics.drawImage(getAssets().getImage("TECH_CONNECT5"), 2160, 290);
+	}
+
+	private void drawTechnologies() {
+        fertilizer.draw(techGraphics, new Point (0,80));
+        wheelbarrow.draw(techGraphics, new Point(0, 190));
+        tent.draw(techGraphics, new Point(0, 300));
+        ironMining.draw(techGraphics, new Point(275, 135));
+        bed.draw(techGraphics, new Point(275, 245));
+        housing.draw(techGraphics, new Point(550, 190));
+        draftHorse.draw(techGraphics, new Point(825, 80));
+        irrigation.draw(techGraphics, new Point(825, 190));
+        steamPower.draw(techGraphics, new Point(825, 300));
+        militia.draw(techGraphics, new Point(1100, 80));
+        pesticides.draw(techGraphics, new Point(1100, 190));
+        steelMining.draw(techGraphics, new Point(1100, 300));
+        barracks.draw(techGraphics, new Point(1375, 80));
+        beer.draw(techGraphics, new Point(1375, 190));
+        blastFurnace.draw(techGraphics, new Point(1375, 300));
+        roads.draw(techGraphics, new Point(1650, 135));
+        vodka.draw(techGraphics, new Point(1650, 245));
+        urbanPlanning.draw(techGraphics, new Point(1925, 135));
+        windPower.draw(techGraphics, new Point(1925, 245));
+        militaryAcademy.draw(techGraphics, new Point(2200, 135));
+        nuclearPower.draw(techGraphics, new Point(2200, 245));
+	}
+	
+	private void paneClicked(double x, double y) {
+		if (pointInPane(14, 87, x, y)) {
```

Comment:
The panes as we have them now are just images. I thought about doing buttons, but then that causes problems in and of itself.